codalab/lib/worksheet_util.py

```diff
@@ -443,7 +443,9 @@ def interpret_file_genpath(client, target_cache, bundle_uuid, genpath, post):
                 contents = client.head_target(target, MAX_LINES)
                 contents = map(base64.b64decode, contents)
 
-            if all('\t' in x for x in contents):
+            if isinstance(contents, list) and len(contents) == 0:
```

Comment:
`contents` should be guaranteed to be a list here, so no need to check its class.